CHANGELOG.md

```diff
@@ -0,0 +1,330 @@
+# _fastlane_ CHANGELOG
+
+---
+
+When contributing to fastlane, don't forget to add an entry to this CHANGELOG.md file by copy/pasting and adapting this entry template:
+
+```
+* Describe your change here. End it with a full-stop and 2 spaces.  
```

Comment:
@milch Ending a line with two spaces in Markdown will render as a line break inside the same paragraph without creating a new paragraph

https://daringfireball.net/projects/markdown/syntax#p

> A paragraph is simply one or more consecutive lines of text, separated by one or more blank lines. (A blank line is any line that looks like a blank line — a line containing nothing but spaces or tabs is considered blank.) Normal paragraphs should not be indented with spaces or tabs.
>
> The implication of the “one or more consecutive lines of text” rule is that Markdown supports “hard-wrapped” text paragraphs. This differs significantly from most other text-to-HTML formatters (including Movable Type’s “Convert Line Breaks” option) which translate every line break character in a paragraph into a `<br />` tag.
>
> **When you do want to insert a `<br />` break tag using Markdown, you end a line with two or more spaces, then type return.**

This allows the author & PR links to be in a dedicated line clearly separated from the entry text, but still attached to the same bullet point.